src/components/10-atoms/checkbox/index.js

```diff
@@ -83,7 +83,12 @@ class AXACheckbox extends NoShadowDOM {
     const childWrapper = document.createElement('axa-text');
     childWrapper.variant = 'size-3';
     childWrapper.className = 'a-checkbox__children-inline';
-    childWrapper.innerHTML = this.innerHTML;
+
+    // Clone live DOM Node so that we dont have to use mutation obersver which causes a
+    // deadlock because we dont have shadow dom here and it would infintily update - render - update
+    // We will set content of live dome AFTER render is done in order that dom nodes are aligned.
+    // Only needed if we have childs.
```

Comment:
// Clone live DOM node, so that we do not have to use mutation observer, which causes a
// deadlock, because we do not use shadow dom here and it would infinitely update - render - update.
// We will set the content of live DOM AFTER render is done, in order that DOM nodes are aligned.
// Only needed if we there are children.

^I adapted some typos and and wordings. See here: https://www.diffchecker.com/bkdNZPE2

Reply:
// Clone live DOM node, so that we do not have to use mutation observer. Because we do not use shadow DOM here, it 
// would infinitely update - render - update.
// Only needed if we there are children.